RLTSQUARE/Unit3/Helper/Data/Data.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace RLTSquare\Unit3\Helper\Data;
+
+use Magento\Framework\App\Helper\AbstractHelper;
+use Magento\Store\Model\ScopeInterface;
+
+class Data extends AbstractHelper
+{
+    /**
+     * @param $config_path
+     * @return mixed
+     */
+    public function getConfig($config_path): mixed
```

Comment:
Good job on declare(strict_types=1); plus actually defining function return types ... however, you could have also define funtion argument return type, eg. public function getConfig(string $config_path): string